modules/jtframe/devops/build-dockers.sh

```diff
@@ -1,14 +1,84 @@
-#!/bin/bash
-docker image build --file jtcore-base.df --tag jotego/jtcore-base .
-docker image build --file jtcore13.df --tag jotego/jtcore13 /opt/altera
-docker image build --file jtcore17.df --tag jotego/jtcore17 /opt/intelFPGA_lite
-docker image build --file jtcore20.df --tag jotego/jtcore20 /opt/intelFPGA_lite
-docker image build --file linter.df --tag jotego/linter .
-
-# Push the images to dockerhub
-docker login
-docker push jotego/jtcore-base:latest
-docker push jotego/jtcore13:latest
-docker push jotego/jtcore17:latest
-docker push jotego/jtcore20:latest
-docker push jotego/linter:latest
+#!/bin/bash -e
+
+BUILDER_NAME="jotego-builder"
+PLATFORMS="linux/amd64,linux/arm64"
+
+main(){
+    SUCCESS=()
+    FAIL=()
+
+    parse_args $*
+
+    if $PUSH_IMAGES; then
+        prepare_builder $BUILDER_NAME
+        docker login
+    fi
+
+    build "jtcore-base" "." $PUSH_IMAGES $PLATFORMS
+    build "jtcore13" "/opt/altera" $PUSH_IMAGES $PLATFORMS
+    build "jtcore17" "/opt/intelFPGA_lite" $PUSH_IMAGES $PLATFORMS
+    build "jtcore20" "/opt/intelFPGA_lite" $PUSH_IMAGES $PLATFORMS
+    build "linter" "." $PUSH_IMAGES $PLATFORMS
+    build "simulator" "." $PUSH_IMAGES $PLATFORMS
```

Comment:
Good! If you align the arguments by columns, it will be even easier to read. If `$PUSH_IMAGES` and `$PLATFORMS` are global, you can skip passing them.